rubyric/02_getting_started.rst

```diff
@@ -24,6 +24,22 @@ a new menu item in A+ choose Edit course and Menu and press Add new menu item.
 You can also choose to use Rubyric through Haka login. In that case you don't
 need to add Rubyric to menu.
 
+.. important::
+
+  If you want to test Rubyric locally, you must read the
+  `Official Rubyric documentation <https://github.com/Aalto-LeTech/rubyric/blob/master/doc/rubyric.md#connect-local-rubyric-to-a-course-in-docker-optional>`_
+  and follow the instructions on how to use a local copy of Rubyric along with A+.
```

Comment:
Running Rubyric locally would be much easier if we created a Docker container for it, but that has not been done.

Reply:
I tried but it was slightly difficult, maybe I will try again in the future.